src/extractor/scripting_environment_lua.cpp

```diff
@@ -276,6 +288,71 @@ void Sol2ScriptingEnvironment::InitContext(LuaScriptingContext &context)
         "version",
         &osmium::Way::version);
 
+    struct RelationMemberWrap
```

Comment:
Why not  to expose `osmium::RelationMember` directly to avoid copying role strings?

Reply:
Because it has deleted a lot of constructors and operators. So it doesn't compiles with sol.
I've tried for example:
```cpp
context.state.new_usertype<osmium::Relation>("Relation",
                                                 "get_value_by_key",
                                                 &get_value_by_key<osmium::Relation>,
                                                 "id",
                                                 &osmium::Relation::id,
                                                 "version",
                                                 &osmium::Relation::version,
                                                 "members",
                                                 [](const osmium::Relation &rel) {
                                                     return sol::as_table(rel.members());
                                                 });
```

Reply:
The wrapper can be removed `return sol::as_table(rel.members());` should work well